deployment/terraform/efs.go

```diff
@@ -0,0 +1,87 @@
+package terraform
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+const (
+	efsDirectory  = "/mnt/mattermost-data"
+	svcScript     = "/opt/mount.sh"
+	efcServiceDir = "/lib/systemd/system/efs-mount.service"
+	efsService    = `[Unit]
+Description=Script to run after fstab to change ownership of mount directory
+After=local-fs.target
+
+[Service]
+Type=simple
+ExecStart=/bin/bash -c "/opt/mount.sh"
+
+
+[Install]
+WantedBy=multi-user.target
+`
+)
+
+func (t *Terraform) setupEFS(extAgent *ssh.ExtAgent) error {
+	for _, val := range t.output.Instances {
+		sshc, err := extAgent.NewClient(val.PublicIP)
```

Comment:
Let's take https://github.com/mattermost/mattermost-load-test-ng/pull/881 into account in case it's merged before this one. cc/@fmartingr